easybuild/easyblocks/i/ipp.py

```diff
@@ -83,20 +84,36 @@ def sanity_check_step(self):
         shlib_ext = get_shared_lib_ext()
 
         if LooseVersion(self.version) < LooseVersion('8.0'):
-            dirs = ['compiler/lib/intel64', 'ipp/bin', 'ipp/include',
-                    'ipp/interfaces/data-compression', 'ipp/tools/intel64']
+            dirs = [
+                os.path.join('compiler', 'lib', 'intel64'),
+                os.path.join('ipp', 'bin'),
+                os.path.join('ipp', 'include'),
+                os.path.join('ipp', 'interfaces', 'data-compression'),
+                os.path.join('ipp', 'tools', 'intel64')
+            ]
         elif LooseVersion(self.version) >= LooseVersion('9.0'):
-            dirs = ['ipp/bin', 'ipp/include', 'ipp/tools/intel64']
+            dirs = [
+                os.path.join('ipp', 'bin'),
+                os.path.join('ipp', 'include'),
+                os.path.join('ipp', 'tools', 'intel64')
+            ]
         else:
-            dirs = ['composerxe/lib/intel64', 'ipp/bin', 'ipp/include',
-                    'ipp/tools/intel64']
+            dirs = [
+                os.path.join('composerxe', 'lib', 'intel64'),
+                os.path.join('ipp', 'bin'),
+                os.path.join('ipp', 'include'),
+                os.path.join('ipp', 'tools', 'intel64')
+            ]
```

Comment:
I would write this slightly different. Something like
dirs = []
ipp_dir_list = ['bin', 'include', os.path.join('tools', 'intel64')]
if LooseVersion ...
  add the version specific ones to dirs and ipp_dir_list
dirs.extend(os.path.join('ipp', d)) for d in ipp_dir_list
(At least i think it is extend one should use, but i usually never get that right)